src/components/InventoryDetail/AppInfo.js

```diff
@@ -54,11 +58,7 @@ AppInfo.propTypes = {
   }),
 };
 
-/**
- * This component can be removed once all apps migrate to componentMapper and activeApp.
- * @param { componentMapper, activeApp } props.
- * @returns AppInfo component
- */
+// TODO: This component can be removed once all apps migrate to componentMapper and activeApp
```

Comment:
We could use the `@deprecated` tag here. Just to practice good practice. :) 

https://jsdoc.app/tags-deprecated